Index: bagit-java/src/assembly/bin/bag
===================================================================
--- bagit-java.orig/src/assembly/bin/bag
+++ bagit-java/src/assembly/bin/bag
@@ -33,4 +33,4 @@ APP_HOME="/usr/share/bagit"
 APP_NAME=`basename $PRG`
 DATE=`date "+%Y%m%d-%T"`
 
-"$JAVA_BIN" "$JAVA_OPTS" -Xmx$MAXMEM -classpath $APP_HOME/lib/classworlds-1.1.jar -Dclassworlds.conf=$APP_HOME/bin/$APP_NAME.classworlds.conf -Dapp.home=$APP_HOME -Dlog.timestamp=$DATE -Dversion=${pom.version} org.codehaus.classworlds.Launcher "$@"
+"$JAVA_BIN" "$JAVA_OPTS" -Xmx$MAXMEM -classpath $APP_HOME/lib/classworlds-debian.jar -Dclassworlds.conf=$APP_HOME/bin/$APP_NAME.classworlds.conf -Dapp.home=$APP_HOME -Dlog.timestamp=$DATE -Dversion=4.12.3 org.codehaus.classworlds.Launcher "$@"
