--- fits-ngserver.sh-orig	2018-10-23 20:18:27.621439745 +0200
+++ fits-ngserver.sh	2018-10-23 20:20:45.943777149 +0200
@@ -17,7 +17,7 @@
 	NAILGUN_JAR=$1
 fi
 
-cmd="java -classpath \"$APPCLASSPATH:$NAILGUN_JAR\" com.martiansoftware.nailgun.NGServer"
+cmd="java -Dlog4j.configuration=file:\"$FITS_HOME\"/log4j.properties -classpath \"$APPCLASSPATH:$NAILGUN_JAR\" com.martiansoftware.nailgun.NGServer"
 
 echo "You may now run FITS by typing: ng edu.harvard.hul.ois.fits.Fits [options]" >&2
 
