--- jhove-installer/src/main/scripts/jhove.orig	2019-06-03 13:33:40.692678887 +0200
+++ jhove-installer/src/main/scripts/jhove	2019-06-03 13:34:30.278520121 +0200
@@ -1,5 +1,19 @@
 #!/bin/sh
 
+jar_list=""
+for i in $(find /usr/share/jhove/bin/ -name jhove*jar); do
+     if [ -z "$jar_list" ]
+     then
+          jar_list=${i}
+     else
+          jar_list=${jar_list}:${i}
+     fi
+done
+
+java -cp "${jar_list}" Jhove -c /usr/share/jhove/conf/jhove.conf "${@}"
+
+exit $?
+
 # JHOVE - JSTOR/Harvard Object Validation Environment
 #
 # Copyright 2003-2005 by JSTOR and the President and Fellows of Harvard College
--- jhove-installer/src/main/scripts/jhove-gui.orig	2019-06-03 13:34:10.045768885 +0200
+++ jhove-installer/src/main/scripts/jhove-gui	2019-06-03 13:34:40.150886655 +0200
@@ -1,5 +1,19 @@
 #!/bin/sh
 
+jar_list=""
+for i in $(find /usr/share/jhove/bin/ -name jhove*jar); do
+     if [ -z "$jar_list" ]
+     then
+          jar_list=${i}
+     else
+          jar_list=${jar_list}:${i}
+     fi
+done
+
+java -cp "${jar_list}" Jhove -c /usr/share/jhove/conf/jhove.conf "${@}"
+
+exit $?
+
 # JHOVE - JSTOR/Harvard Object Validation Environment
 #
 # Copyright 2003-2005 by JSTOR and the President and Fellows of Harvard College
