diff -u jhove-installer/src/main/scripts.orig/jhove jhove-installer/src/main/scripts/jhove
--- jhove-installer/src/main/scripts.orig/jhove	2016-03-26 03:04:56.668017880 +0000
+++ jhove-installer/src/main/scripts/jhove	2016-03-26 03:05:34.448981124 +0000
@@ -30,8 +30,8 @@
 # adjust your shell's path or revert to the old way (commented out).
 # Configuration constants:
 
-JHOVE_HOME=%INSTALL_PATH
-JHOVE_VERSION=%APP_VER
+JHOVE_HOME=/usr/share/jhove
+JHOVE_VERSION=1.16.0
 
 #XTRA_JARS=/users/stephen/xercesImpl.jar
 EXTRA_JARS=              # Extra .jar files to add to CLASSPATH
@@ -52,4 +52,4 @@
 #{JAVA} -classpath $CP Jhove $ARGS -x org.apache.xerces.parsers.SAXParser
 #${JAVA} -classpath $CP Jhove $ARGS
 # New way, doesn't require you to use JAVA_HOME.
-java -classpath $CP Jhove $ARGS
\ No newline at end of file
+java -classpath $CP Jhove $ARGS
diff -u jhove-installer/src/main/scripts.orig/jhove-gui jhove-installer/src/main/scripts/jhove-gui
--- jhove-installer/src/main/scripts.orig/jhove-gui	2016-03-26 03:04:56.668017880 +0000
+++ jhove-installer/src/main/scripts/jhove-gui	2016-03-26 03:05:46.641969194 +0000
@@ -30,8 +30,8 @@
 # adjust your shell's path or revert to the old way (commented out).
 # Configuration constants:
 
-JHOVE_HOME=%INSTALL_PATH
-JHOVE_VERSION=%APP_VER
+JHOVE_HOME=/usr/share/jhove
+JHOVE_VERSION=1.16.0
 
 #XTRA_JARS=/users/stephen/xercesImpl.jar
 EXTRA_JARS=              # Extra .jar files to add to CLASSPATH
@@ -52,4 +52,4 @@
 #{JAVA} -classpath $CP Jhove $ARGS -x org.apache.xerces.parsers.SAXParser
 #${JAVA} -classpath $CP Jhove $ARGS
 # New way, doesn't require you to use JAVA_HOME.
-java -classpath $CP JhoveView $ARGS
\ No newline at end of file
+java -classpath $CP JhoveView $ARGS
