From 2319c81b6484ca6249fa956c69f38089edeafd95 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Santiago=20Rodr=C3=ADguez?= <santi@collazo.eu>
Date: Thu, 16 Mar 2017 18:13:51 +0100
Subject: [PATCH] Upgrade path for centos

---
 bin/am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/bin/am b/bin/am
index 6e35be1..9e6c13f 100755
--- a/bin/am
+++ b/bin/am
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-prefix="@PREFIX@"
+prefix="/usr/lib/archivematica/devtools"
 
 case $1 in
 	'')
@@ -13,6 +13,7 @@ case $1 in
 		;;
 	*)
 		export PYTHONPATH=$PYTHONPATH:/usr/share/archivematica/dashboard:/usr/lib/archivematica/archivematicaCommon
+		export PATH=/usr/lib/python2.7/archivematica/MCPClient/bin/:$PATH
 		# respect DJANGO_SETTINGS_MODULE if set in the environment
 		if [ -z "$DJANGO_SETTINGS_MODULE" ]; then
 			export DJANGO_SETTINGS_MODULE="settings.local"
-- 
2.7.4

