Index: MCPServer/init/archivematica-mcp-server.conf
===================================================================
--- MCPServer.orig/init/archivematica-mcp-server.conf
+++ MCPServer/init/archivematica-mcp-server.conf
@@ -22,16 +22,15 @@ start on (started mysql)
 stop on runlevel [016]
 
 env CONF=/etc/archivematica/MCPServer
-env LOCATION=/usr/lib/archivematica/MCPServer/archivematicaMCP.py
+env LOCATION="/usr/share/python/archivematica-mcp-server/bin/python /usr/lib/archivematica/MCPServer/archivematicaMCP.py"
+env PYTHONPATH=/usr/lib/archivematica/archivematicaCommon/:/usr/share/archivematica/dashboard/
+
 
 setuid archivematica
 setgid archivematica
 
 pre-start script
 
-    # Check that $CONF directory exists
-    [ -d $CONF ]
-
     # Wait for Gearman to start - timeout eventually
     for i in $(seq 1 10)
     do
@@ -44,7 +43,8 @@ pre-start script
 end script
 
 script
-
+    # Load environment
+    export $(cat /etc/default/archivematica-mcp-server)
     # Run
     $LOCATION
 
