--- Makefile-orig	2018-10-29 21:57:47.231529852 +0100
+++ Makefile	2018-10-29 21:55:56.081451026 +0100
@@ -13,7 +13,7 @@

 ng: ${SRCDIR}/ng.c
 	@echo "Building ng client.  To build a Windows binary, type 'make ng.exe'"
-	${CC} $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -o ng ${SRCDIR}/ng.c
+	${CC} $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -o ng-nailgun ${SRCDIR}/ng.c

 install: ng
 	install -d ${PREFIX}/bin
@@ -30,5 +30,5 @@
 	@echo "You must remove this manually.  Most users won't have MinGW"
 	@echo "installed - so I'd rather not delete something you can't rebuild."
 	@echo ""
-	rm -f ng
+	rm -f ng-nailgun
 #	rm -f ng.exe
